packages/app/src/cli/validators/extensions/ui.ts

```diff
@@ -1,5 +1,5 @@
 import {UIExtension} from '../../models/app/extensions.js'
 
 export async function validateUIExtensions(extensions: UIExtension[]) {
-  return Promise.all(extensions.map((ext) => ext.preDeployValidation()))
+  return Promise.all(extensions.map((ext) => ext.validate()))
```

Comment:
Why this change? This function is called during deploy so it makes sense to use the `preDeployValidation` no?

Reply:
Yeah, you're right.  This was a mistake, reverted! Thanks :+1: